.github/workflows/desktop.yml

```diff
@@ -274,3 +274,85 @@ jobs:
           path: |
             ~/Library/Logs/DiagnosticReports/*
             /tmp/bins/*
+
+  sanity_check: 
```

Comment:
Let's name this something less ableist. Maybe something like local_integration_tests?

Reply:
Sure